content/en/docs/ibm/deploy/install-kubeflow.md

```diff
@@ -75,21 +75,23 @@ get the best experience from Kubeflow.
 
 6. Set the Block Storage as the default storageclass.
     ```shell
-    kubectl patch storageclass ibmc-block-gold -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"true"}}}'
+    NEW_STORAGE_CLASS=ibmc-block-gold
+    OLD_STORAGE_CLASS=$(kubectl get sc | grep "(default)" | grep -v ${NEW_STORAGE_CLASS} | awk '{ print $1;exit }')
```

Comment:
Do you think this command is better? `kubectl get sc -o jsonpath='{.items[?(@.metadata.annotations.storageclass\\.kubernetes\\.io\/is-default-class=="true")].metadata.name}'` works on all OSes.

Reply:
The jsonpath command doesn't work on my machine, let me see is there a better way to grep it.

Reply:
@shawnzhu I updated the command to use jsonpath